cmds/exp/cpu/cpu9p.go

```diff
@@ -0,0 +1,283 @@
+// Copyright 2018 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/hugelgupf/p9/p9"
+	"github.com/hugelgupf/p9/unimplfs"
+)
+
+// cpu9p is a p9.Attacher.
+type cpu9p struct {
```

Comment:
how is this different from localfs?

can you embed localfs for most of them except where there's a difference?

Reply:
good idea, I'll take a look. I think there were issues but I no longer recall. 

The only other possible problem is that cpu is then dependent on whatever changes are made to localfs over time, which is both good and bad: we pick up fixes transparently, but we might get broken in ways that are not clear b/c we use localfs. 

This exact scenario played out in plan 9 over and over. There was a 9p library (like localfs) and there were little 9p servers in lots of programs, 9p servers being very easy to write. Each model had its own good and bad; neither was unquestionably superior. 
'
But I'll take a look. It would certainly be nice to just embed localfs.

Reply:
actually what might be good is to put that localfs embedding into a different commit I can easily revert if issues arise?